src/librustc/benches/pattern.rs

```diff
@@ -8,7 +8,11 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use test::Bencher;
+#![cfg(test)]
+#![feature(test)]
+#![feature(slice_patterns)]
```

Comment:
is this file the crate root? I'm assuming it's a module, so this attribute does nothing here